docs/release-notes/known-issues.md

```diff
@@ -30,5 +30,148 @@ As a workaround, and to prevent crashes, Beats will ignore any filters provided
 
 % **Resolved**
 % On [Month/Day/Year], this issue was resolved.
+:::
 
-:::
\ No newline at end of file
+:::{dropdown} Beats panic on restart when "restart_on_cert_change" is enabled on Linux
+
+**Details**
+A known issue was discovered where Beats running on Linux with `restart_on_cert_change` enabled panic during a restart. This occurs because the default seccomp policy does not include the `eventfd2` syscall, which is used by Go runtime versions 1.23.0. While the initial launch is successful, subsequent restarts fail as the seccomp policy is already active, blocking the required syscall.
+
+This affects versions v8.16.6+, v8.17.3+, v8.18.0+, v8.19.0+, v9.0.0+, and v9.1.0+.
```

Comment:
@colleenmcginnis, do you think it's good to list all affected versions here and like that or is there a better way to deal with it?

Reply:
This is a tricky one. I think this is fine for now. A related question came up in Slack this week. I'll revisit how we use `applies_to` in known issues across ingest docs after the next round of design updates in docs-builder.